app/package.json

```diff
@@ -51,9 +51,11 @@
     "@types/react-onclickoutside": "6.7.3",
     "babel-plugin-emotion": "10.0.33",
     "babel-plugin-jsx-remove-data-test-id": "2.1.3",
+    "babel-plugin-transform-dynamic-import": "^2.1.0",
     "enzyme": "3.11.0",
     "enzyme-adapter-react-16": "1.15.2",
     "jest-dom": "4.0.0",
+    "jest-next-dynamic": "^1.0.1",
```

Comment:
Please lock versions (remove `^`) of `"babel-plugin-transform-dynamic-import"` and  `"jest-next-dynamic"`